basiclti/basiclti-tool/src/java/org/sakaiproject/blti/tool/LTIAdminTool.java

```diff
@@ -205,6 +434,30 @@ public String buildToolSitePanelContext(VelocityPortlet portlet, Context context
 		context.put("isAdmin",new Boolean(ltiService.isAdmin()) );
 		context.put("getContext",toolManager.getCurrentPlacement().getContext());
 		
+		//put velocity date tool in the context
+		context.put("dateTool", new DateTool());
+		DateFormat df = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT, rb.getLocale());
+		if (df instanceof SimpleDateFormat)
+		{
+			SimpleDateFormat sdf = (SimpleDateFormat)df;
+			context.put("dateToolFormat",sdf.toLocalizedPattern());
```

Comment:
Yep, but why not do:
```
context.put("dateTool", df);
```
then in the velocity file just do the formatting
```
$!dateTool.format($!content.get("created_at"))
```
The reason for using velocity DateTool class is so that Velocity does automatic formatting of dates without you having to explicitly say it should use a formatter, but at the moment this isn't being used so it's just as easy to pass in the Java date formatter.